mlflow/langchain/_langchain_autolog.py

```diff
@@ -42,7 +42,9 @@ def patched_inference(func_name, original, self, *args, **kwargs):
     # NB: Running the original inference with disabling autologging, so we only patch the top-level
     # component and avoid duplicate logging for child components.
     def _invoke(self, *args, **kwargs):
-        with disable_autologging():
+        # Original function might launch multiple worker threads, to avoid breaking change,
+        # we need to disable autologging globally.
+        with disable_autologging(is_global=True):
```

Comment:
Update: use globally disablement here to avoid breaking change.

Reply:
Great catch! Minor: can we replace "to avoid breaking change" to the actual issue that would be caused when not disabling globally? For ppl who saw this comment outside PR, it is not clear what will be a "breaking change".